assets/index.html

```diff
@@ -519,8 +513,7 @@ <h2 class="value">
     </div>
   </div>
 </footer>
-
-<script src="js/app.js?<<.Tag>>"></script>
```

Comment:
.Tag brauchen wir damit iOS Web Apps sich refreshen können. Egtl. auch analog fürs Icon, ggf. mittels Hash im Backend?

Reply:
Das explizite Tagging ist mit Parcel nicht mehr notwendig da er automatisch alle Assets mit Content-Hashes versieht.

Reply:
In der generierten `dist/assets/index.html` siehst du dann auch die angepassten Referenzen.
`<script src="/dist/app.f170d70e.js"></script>`

Reply:
Ahhh, nice 👍🏻. Es könnte aber sein, dass embed das nicht versteht, ich glaube es „sieht“ neue Dateien nicht. Muss ich probieren. Falls das der Fall sein sollte müssten wir auf Go 1.16. warten.

Ist der Build deterministisch, also werden immer die gleichen Referenzen bei gleichem Content erzeugt?

Reply:
Ja, eigentlich sollte der Assets Build deterministisch sein. Lokal auf meiner Maschine war das auch immer so. Im vorherigen Travis Lauf scheint sich aber die assets.go geändert zu haben. Daher hab ich sie auch ins ignore gepackt.
Bin mir nicht sicher welche Node/NPM Version in Travis gezogen wird oder das OS noch eine Rolle spielt.

Was meinst du mit dem `clean` Check und dass "embed" das nicht versteht?

Reply:
> Im vorherigen Travis Lauf scheint sich aber die assets.go geändert zu haben.

Die Generierung der assets.go ist deterministisch- dann liegt das Problem vorher. 

> Was meinst du mit dem clean Check und dass "embed" das nicht versteht?

Ich meine https://github.com/andig/evcc/blob/master/.travis.yml#L31. Da wird gecheckt ob notwendige generierte Artefakte sauber im git eingecheckt sind, d.h. also ob beim assets generieren wieder das gleiche Ergebnis raus kommt.

Reply:
Ich hab jetzt noch mal sichergestellt, dass gehashte Assets aus vorherigen Builds auch abgeräumt werden. Wenn Travis den Workspace nicht abräumt kann es gut sein, das das auch der Grund war warum die assets.go beim zweiten Durchlauf modifiziert wurde.